app/phonebook.py

```diff
@@ -0,0 +1,16 @@
+phonebook = {}
+
+def addContact(firstname, lastname, number):
+    return {"message": "Contact created successfully"} 
+
+
+def updateContact(contact1, contact2): 
+    return {"message": "Contact updated successfully"}

```

Comment:
@WhitneyOgutu You are only returning error messages.. I cannot see the logic of adding, updating, deleting and viewing contacts